README.md

```diff
@@ -482,9 +489,25 @@ sym::Optimizer<double> optimizer(
 
 sym::Values<double> values;
 for (int i = 0; i < num_poses; ++i) {
-    values.Set({'P', i}, sym::Pose2::Identity());
+    values.Set({'P', i}, sym::Pose2d::Identity());
 }
-// ... (initialize all keys)
+
+// Set additional values
+values.Set({'L', 0}, Eigen::Vector2d(-2, 2));
+values.Set({'L', 1}, Eigen::Vector2d(1, -3));
+values.Set({'L', 2}, Eigen::Vector2d(5, 2));
+values.Set({'d', 0}, 1.7);
+values.Set({'d', 1}, 1.4);
+values.Set({'a', 0, 0}, 55 * M_PI / 180);
+values.Set({'a', 0, 1}, 245 * M_PI / 180);
+values.Set({'a', 0, 2}, -35 * M_PI / 180);
+values.Set({'a', 1, 0}, 95 * M_PI / 180);
+values.Set({'a', 1, 1}, 220 * M_PI / 180);
+values.Set({'a', 1, 2}, -20 * M_PI / 180);
+values.Set({'a', 2, 0}, 125 * M_PI / 180);
+values.Set({'a', 2, 1}, 220 * M_PI / 180);
+values.Set({'a', 2, 2}, -20 * M_PI / 180);
+values.Set('e', sym::kDefaultEpsilond);
 
 // Optimize!
```

Comment:
I do think it's worth having a fully complete C++ version in examples/robot_2d_triangulation

Reply:
@bradley-solliday-skydio is going to do this

Reply:
#167